specs/LocaleRegion.md

```diff
@@ -0,0 +1,107 @@
+LocaleRegion
+===
+
+# Background
+Developers have requested access to programmatically choose the specific locale/region to use in WebView2. 
+The locale/region updates the JavaScript region (objects in the ECMA standard) which gets reflected as
+variations in time and date formats. Currently, the locale is by default set to the same value as the 
+display language. 
+
+You can use the `LocaleRegion` API to update the locale value individually from the display
+language.
+
+# Description
+* The intended value for `LocaleRegion` is in the format of `language[-country]` where `language` is the 
+2-letter code from [ISO 639](https://www.iso.org/iso-639-language-codes.html) and `country` is the 
+2-letter code from [ISO 3166](https://www.iso.org/standard/72482.html).
+* By default, the `LocaleRegion` will be set as the value for the Limited option in the browser.
+That means that if the OS region and the display language share the same language code, 
+like 'en-GB' and 'en-US', the value will be set to the OS region.

```

Comment:
It would be good to have a table of different options and the result. Something like the following (but with more rows) in the /// ref doc comments

OS Region | WebView2 Language | Default WebView2 LocaleRegion
---|---|---
 en-GB | en-US | en-GB
 ... | ... | ...

Reply:
Got it. Will add this as part of the experimental PR